blog/arquillian-extension-drone-2.4.1.textile

```diff
@@ -0,0 +1,15 @@
+---
+date: 2017-08-24
+author: matousjobanek
+---
+
+!(graphic pull-right)/images/blog/drone_400.png!
+
+h3. Authenticated GitHub requests
+
+In case of downloading webdrivers from GitHub, you may reach the rate limit for unauthenticated requests (60 per hour). To avoid this, there had been implemented several caching mechanisms, but in some cases (larger CI matrix jobs) it doesn't have to suffice.
+
+Because of these problems, in this release, we have introduced support for authenticated GitHub requests. In the case of authenticated connections the rate limit is much higher, so it should be almost impossible to reach the limit. To use it you need to specify your GitHub username and secret token in your @arquillian.xml@ file:
+
+bc(prettify).. <property name="githubUsername">username</property>
+<property name="githubToken">secretToken</property>
```

Comment:
Maybe note, that you can also use properties (showing how), so you don't have to comprise your credential in the publicly available repositories.